op-deployer/pkg/deployer/pipeline/opchain.go

```diff
@@ -114,23 +115,26 @@ func makeDCI(intent *state.Intent, thisIntent *state.ChainIntent, chainID common
 }
 
 func makeChainState(chainID common.Hash, dco opcm.DeployOPChainOutput) *state.ChainState {
+	opChainContracts := addresses.OpChainContracts{}
```

Comment:
Tiny thing but why not set all these fields in the constructor here instead of setting them one by one?

Reply:
This way is a bit simpler because `OpChainContracts` is a parent struct with several embedded child structs. So I can access the address fields directly on the parent struct. But if I try to set the fields in the constructor, I have to explicitly specify the child struct each address belongs to